Wordle.Api/Wordle.Api/Controllers/WordController.cs

```diff
@@ -36,9 +48,31 @@ public async Task<Word> AddWord(string newWord, bool isCommon)
         }
 
         [HttpPost("AddWordFromBody")]
+        [Authorize(Policy = "NewPolicy")]
         public async Task<Word> AddWordFromBody([FromBody] WordDto word)
         {
             return await _wordService.AddWord(word.Text, word.IsCommon);
         }
+
+        [HttpPost("DropWord")]
+        [Authorize(Policy = "NewPolicy")]
+        public async Task<bool> DropWord(string word) {
+            return await _wordService.DropWord(word); 
+        
+        }
+
+        [HttpPost("FlipIsCommon")]
+        [Authorize]
```

Comment:
Oddly, this does not let me flip the word if I am logged in as just an admin

Reply:
Hmm, good to know. Thanks Meg.